inst/qml/qml_components/BayesianMetaAnalysisInference.qml

```diff
@@ -43,25 +43,49 @@ Section
 			{
 				value: 				"fixed"
 				label: 				qsTr("Fixed effects")
+				onCheckedChanged:	{
+					bayesianMetaAnalysisAdvanced.priorH0FEValue = 0.5
+					bayesianMetaAnalysisAdvanced.priorH1FEValue = 0.5
+					bayesianMetaAnalysisAdvanced.priorH0REValue = 0
+					bayesianMetaAnalysisAdvanced.priorH1REValue = 0
+				}
```

Comment:
Oh I see the logic is already implemented in BayesianMetaAnalysisAdvanced.qml, so all this onCheckedChanged can be removed.
I see one difference in the logic: if the value is `constrainedRandom`, it has here the same value as `averaging` (0.25 for every properties), but in the BayesianMetaAnalysisAdvanced, as it will take the default value (0).

Reply:
I did not change this however, it was added in the rebase